w3-aggregation.md

```diff
@@ -132,7 +132,7 @@ A Storefront principal can invoke a capabilty to offer an aggregate that is read
       "offer": { "/": "bafy...many-cars" }, /* dag-cbor CID with offer content */
       "piece": {
```

Comment:
should this be `pieceProof` or even just `proof`? See rationale on other review comment

Reply:
> Actually can we just call this an aggregate instead ?

we previously went back from `aggregate` to `piece` given they by the end of the day are the same. Have `height` and `cid`, and perhaps in the future they could be encoded in the CID as you say.